docs/source/guide/storage.md

```diff
@@ -309,15 +309,14 @@ Add these variables to your environment setup:
 Without these settings, Local storage and URLs in tasks that point to local files won't work. Keep in mind that serving data from the local file system can be a **security risk**. See [Set environment variables](start.html#Set_environment_variables) for more about using environment variables.
 
 
-
 ### Set up connection in the Label Studio UI
 In the Label Studio UI, do the following to set up the connection:
 
 1. Open Label Studio in your web browser.
 2. For a specific project, open **Settings > Cloud Storage**.
 3. Click **Add Source Storage**.  
 4. In the dialog box that appears, select **Local Files** as the storage type. 
-5. Specify **Local path** directory with your files. The local path must include the LOCAL_FILES_DOCUMENT_ROOT in the beginning. For example, if `LOCAL_FILES_DOCUMENT_ROOT=/home/user`, then your local path must be `/home/user/dataset1`.
+5. Specify **Local path** directory with your files. The local path must be an absolute path and include the LOCAL_FILES_DOCUMENT_ROOT. For example, if `LOCAL_FILES_DOCUMENT_ROOT=/home/user`, then your local path must be `/home/user/dataset1`. For more about that environment variable, see [Run Label Studio on Docker and use local storage](start.html#Run_Label_Studio_on_Docker_and_use_local_storage)
```

Comment:
Is 1 at the end of `dataset1` is a typo, or was it intentional?

Reply:
It's intentional, also used in Tasks with local storage file references section